cli/utils/index.ts

```diff
@@ -43,6 +43,13 @@ export const assertShellResult = (result: ShellString, errMsg: string) => {
   }
 }
 
+export const assertFindings = (findings: Finding[]) => {
+  const serializedFindings = JSON.stringify(findings).length;
+
+  if(serializedFindings > 50000) throw Error(`Cannot return more than 50kB of findings per request (received ${findings.length})`)
```

Comment:
for this assertion, should we print out the byte size of the received response in the error? (instead of findings.length)

Reply:
Updated to show the byte size